examples/sn-dms-demo/src/assets/picker.ts

```diff
@@ -131,18 +131,18 @@ export const pickerTheme = createMuiTheme({
           boxShadow: 'none',
         },
       },
-      '&$disabled': {
-        background: '#fff',
-        color: '#016D9E',
-        boxShadow: 'none',
-        '&.disabled-mobile-button': {
-          [theme.breakpoints.down('md')]: {
-            background: '#fff',
-            color: '#016D9E',
-            opacity: 0.25,
-          },
-        },
-      },
+      // '&$disabled': {
```

Comment:
Is this commented out code left there intentionally? If not I think there is no need to add commented out code to source control.